.circleci/config.yml

```diff
@@ -142,7 +142,7 @@ commands:
 jobs:
   wp-desktop-assets:
     docker:
-      - image: circleci/node:12.16.2-browsers
+      - image: circleci/node:16.17.0-browsers
```

Comment:
While this is a big update, the workflow below already uses `nvm` to update Node to a more recent version, so it wasn't using 12.16 anyways.

Reply:
Could you elaborate?

Also, do you know why this is in a `.circleci` directory? Do we still use CircleCI?

Reply:
Yeah, as far as I know, we use circleci to build the desktop app. (Not the web app)

Reply:
Yeah, I doubt we have Windows or macOS server environments in TeamCity 😅

Reply:
Alright, thanks for clarifying folks!